### DISCUSSION TITLE
TB 115 breaks Sendlater (and a host of other add-ons)

### DISCUSSION THREAD

**Discussion by rgrun1** [Extended-Thunder/send-later#552] [General]
I had an update to the big turd known as v115 this morning which broke Send-later

**Comment by telrod11** [comment]
Yeah, this is one of my got to add ons for TB.   Had to drop back to 102.

Will this get updated, or is this the end for SL?

**Comment by NathanVeenstra** [comment]
Is this due to a TB update? I thought it might be because I use Proton Mail. Reading this, I'm getting my hopes up that it's not Proton and it might be fixed eventually ...

@telrod11 could you help a non-techie how to drop back to 102 please?

**Reply by telrod11** [reply]
Unfortunately, there is no path to regress, as far as I know.    Were you and I to have chosen another directory to install we could have easily gone back, but I stupidly just let it update, with no thought of how I might miss SendLater

**Reply by jikamens** [reply]
You can certainly download and use both previous Thunderbird versions and previous Send Later versions, so I'm not sure exactly what you're saying here. Can you clarify?

**Reply by telrod11** [reply]
@jikamens

Not sure if this was directed at me, or not, but what I was saying is if you let the 115 update *overwrite* 102, then you have to start over.

**Comment by NathanVeenstra** [comment]
Wait. I am on 102.13.0, could it be that this release has already broken Send Later? I'm confused now, especially as I am such a non-tech guy and I have no clue how to even discover what the problem is ...

**Comment by jikamens** [comment]
Send Later should work in 102.13.0, though there are a few intermittent bugs in Send Later which impact both that TB version and others.

We're working on getting Send Later uplifted to TB115. We'll get there.

**Reply by telrod11** [reply]
It seems to work flawlessly for my on 102.13.0.    I mean, I'm not sending 50 email a day with  it, but have no issues with the few I send a day.

**Comment by NathanVeenstra** [comment]
I haven't been able to use Send Later for months now, it's been so helpful before.

**Comment by jikamens** [comment]
>I haven't been able to use Send Later for months now, it's been so helpful before.
If you're not using TB115, then we should discuss that elsewhere.
If you've already posted about it elsewhere (discussion or issue), feel free to post a link here and I'll follow up with you. Otherwise, please open a new issue or discussion about the problem you are having and we will try to get it fixed.

**Reply by NathanVeenstra** [reply]
I posted it here: https://github.com/Extended-Thunder/send-later/discussions/562

And thanks for your replies so far. It made me decide to become a Patreon ;-)

**Comment by jikamens** [comment]
There is now a beta release for TB115. Please see details [here](https://github.com/Extended-Thunder/send-later/releases/tag/v10.0.0).

**Reply by alexandrokatayama** [reply]
It works on 115.1.0.

**Reply by telrod11** [reply]
Not a power user of SL, but it seems to work just fine for me too on TB 115.

Thanks for all the hard work Dev!!
